app/views/merchant/invoices/show.html.erb

```diff
@@ -5,3 +5,11 @@
 <p>Invoice created on: <%= @invoice.format_date %></p>
 <p>Customer Name: <%= @customer.first_name %> <%= @customer.last_name %>
 
+<h3>Invoice Items</h3>
+<% @invoice.invoice_items.each do |invoice_item|%>
+  <p><%= invoice_item.item.name %></p>
+  <p><%= invoice_item.quantity %></p>
+  <p><%= invoice_item.unit_price %></p>
```

Comment:
I have a way we can refactor this to show the unit price in $'s, formatted nicely, that we can implement later.